run/events-pubsub/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Google LLC. All rights reserved.
```

Comment:
Update year

Reply:
Sure. Although OSPO states to not change the year if the file was copied in go/copyright.